src/official-account/official-account.ts

```diff
@@ -154,7 +160,14 @@ class OfficialAccount extends EventEmitter {
      *  "expires_in":7200
      * }
      */
-    const ret = await this.simpleUnirest
+
+    let simpleUnirest: SimpleUnirest = this.simpleUnirest
```

Comment:
If `accessTokenProxyUrl` has been configured, it will refresh the `accessToken` from the Proxy Service. 
This is the easiest way to make `accessToken` centralized.